test-ts/truefi/LoanToken.test.ts

```diff
@@ -110,6 +110,46 @@ describe('LoanToken', () => {
     })
   })
 
+  describe('Reclaim', () => {
+    beforeEach(async () => {
+      await loanToken.fund()
+      await loanToken.connect(borrower).withdraw(borrower.address)
+    })
+
+    it('can only be called by lender', async () => {
+      await payback(borrower, 1000)
+      await timeTravel(provider, monthInSeconds * 12)
+      await loanToken.close()
+      await expect(loanToken.connect(other).reclaim(1000))
+        .to.be.revertedWith('LoanToken: This can be performed only by lender')
+    })
+
+    it('can only be called when status is settled or defaulted', async () => {
+      await payback(borrower, 1000)
+      await expect(loanToken.reclaim(1000))
+        .to.be.revertedWith('LoanToken: Current status should be Settled or Defaulted')
+    })
```

Comment:
I'm missing a test with reclaim on defaulted loan. Now somebody can accidentally change enum order and tests wouldn't catch that

Reply:
sorry, this comment applies to TrueLender tests